src/github.com/getlantern/flashlight/ui/ui.go

```diff
@@ -0,0 +1,72 @@
+package ui
+
+import (
+	"fmt"
+	"net"
+	"net/http"
+	"os"
+	"path"
+	"path/filepath"
+
+	"github.com/getlantern/golog"
+	"github.com/getlantern/tarfs"
+	"github.com/skratchdot/open-golang/open"
+)
+
+const ()
+
+var (
+	log = golog.LoggerFor("ui")
+
+	localResourcesPath = filepath.Join(os.Getenv("GOPATH"), "src/github.com/getlantern/ui/app")
```

Comment:
The code I had before actually looked for the ui directory relative to the source file of the function being invoked which conveniently handled most use cases. It used [runtme.Caller](http://golang.org/pkg/runtime/#Caller)